modules/migration-rolling-back-migration-manually.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/troubleshooting-3-4.adoc
+// * migration_toolkit_for_containers/troubleshooting-mtc
+
+[id="migration-rolling-back-migration-manually_{context}"]
+= Rolling back a migration manually
+
+You can roll back a failed migration manually by deleting the `stage` pods and unquiescing the application.
+
+If you run the same migration plan successfully, the resources from the failed migration are deleted automatically.
+
+[NOTE]
+====
+The following resources remain after a failed direct volume migration (DVM):
+
+* Config maps (source and destination clusters)
+* `Secret` CRs (source and destination clusters)
+* `Rsync` CRs (source cluster)
+
+These resources do not affect rollback. You can delete them manually.
+====
+
+.Procedure
+
+. Delete the `stage` pods on all clusters:
++
+[source,terminal]
+----
+$ oc delete $(oc get pods -l migration.openshift.io/is-stage-pod -n <namespace>) <1>
```

Comment:
Is this being enforced? Most of the openshift modules do not have italics. IMO, this style should be updated globally.

Reply:
@jeana-redhat we don't follow that SSG guidance of italicizing replaceable values yet. This is something we'll have to bring up to the team and do a sweep, once we're ready to make the change.

If it would help, I can add something to the peer review checklist about replaceable value format, and mentioning that we don't italicize yet, even though that's what the SSG currently recommends.

Reply:
Oh, my mistake - I thought we had discussed this and were doing it already. Thanks!